### DISCUSSION TITLE
Windows 8 driver issue while using v86

### DISCUSSION THREAD

**Discussion by RenaultAvantime** [copy/v86#998] [General]
I was wondering how run Windows 8 in v86 because the newest windows option is 2000, and I do not know what to download and how it works.

**Comment by SuperMaxusa** [comment]
see #930 and #852

**Comment by RenaultAvantime** [comment]
I am kind of stupid and don't know where to get windows 8 build 8133, and how to use it. Help?

**Comment by RenaultAvantime** [comment]
I got the iso file but this error showed up:
![screencapture-copy-sh-v86-2024-02-28-15_57_09](https://github.com/copy/v86/assets/131399282/4bf973e0-cbbb-48ba-8729-602e169e35d8)

**Comment by RenaultAvantime** [comment]
![screencapture-copy-sh-v86-2024-02-28-16_21_09](https://github.com/copy/v86/assets/131399282/ab8434ac-9207-4634-8a66-a8f7572c9548)
How to get past this or fix it?

**Comment by spetterman66** [comment]
sir, you're not supposed to install it inside the VM, **you are supposed to run it in qemu and use the resulting disk image in v86**

**Reply by RenaultAvantime** [reply]
Ok, thanks!
